src/shared_modules/keystore/src/keyStore.cpp

```diff
@@ -40,6 +40,38 @@ void Keystore::put(const std::string& columnFamily, const std::string& key, cons
     }
 }
 
-void Keystore::get(const std::string& columnFamily, const std::string& key, rocksdb::PinnableSlice& value)
+void Keystore::get(const std::string& columnFamily, const std::string& key, std::string& value)
 {
+    std::string encryptedValue;
+
+    try
+    {        
+        // Get from DB
+        Utils::RocksDBWrapper keystoreDB = Utils::RocksDBWrapper(DATABASE_PATH, false);
```

Comment:
Is it possible that keystorDB not to be valid? Can that be validated before use?

Reply:
The object cannot be invalid, if any error occurred during the constructor execution, the try-catch will handle it.